docs/content/preview/drivers-orms/go/pgx.md

```diff
@@ -52,7 +50,7 @@ import (
 )
 ```
 
-### Step 2: Connect to YugabyteDB database
+### Step 2: Set up the database connection
 
 Go applications can connect to the YugabyteDB database using the `pgx.Connect()` function. The `pgx` package includes all the common functions or structs required for working with YugabyteDB.
 
```

Comment:
Under SSL are there defaults for those parameters? Also SSLROOTCERT is the path
See Java example

Reply:
Linking to the table listing the default and supported modes and will make the other change for the SSLROOTCERT description to path.